src/game/client/ui.cpp

```diff
@@ -63,9 +63,9 @@ bool CUI::MouseInsideClip() const
 
 void CUI::ConvertMouseMove(float *x, float *y) const
 {
-	float Fac = (float)(m_pConfig->m_UiMousesens)/m_pConfig->m_InpMousesens;
-	*x = *x*Fac;
-	*y = *y*Fac;
+	float Fac = m_pConfig->m_UiMousesens/100.0f;
```

Comment:
This would make the ui mouse sens depend on the the ingame mouse sens which it shouldn't be.

Reply:
Otherwise joystick UI sense depends on this ratio and the menus are basically unuseable with a joystick. I reverted it and will open a new issue, as this requires differentiating between the joystick and mouse movement at basically every layer of the engine/game.